.gitignore

```
@@ -11,3 +11,4 @@ coverage
 *.log
 .env
 TODO
+/upload/
```

`Multer` сам не создаст папку `/uploads/`, поэтому лучше игнорировать только содержания папке.

```suggestion
/upload/*
```